openedx/core/djangoapps/edx_global_analytics/tests/test_utils.py

```diff
@@ -0,0 +1,117 @@
+"""
+Tests for edx_global_analytics application helper functions aka utils.
+"""
+
+import datetime
+
+from django.test import TestCase
+from django.utils import timezone
+
+from django.db.models import Q
+from django_countries.fields import Country
+
+from student.models import UserProfile
+from student.tests.factories import UserFactory
+
+from ..utils import fetch_instance_information, cache_instance_data
+
+
+class TestStudentsAmountPerParticularPeriod(TestCase):
+    """
+    Tests cover all methods, that have a deal with statistics calculation.
+    """
+
```

Comment:
I will fix unnecessary indent in next commits with Jitter implementation. Don`t pay attention to this.